modules/sno-du-accelerating-container-startup.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-accelerating-container-startup_{context}"]
+= Accelerating container startup
+
+
```

Comment:
@imiller0 This is one of the new topics. Please provide an intro sentence here - basically, why they are using this procedure.